lib/ansible/modules/cloud/amazon/ecs_attribute.py

```diff
@@ -0,0 +1,290 @@
+#!/usr/bin/python
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'version': '1.0'}
+
+DOCUMENTATION = '''
+---
+module: ecs_attribute
+short_description: manage ecs attributes
+description:
+    - Create, update or delete ECS container instance attributes.
+version_added: "2.3"
+author: Andrej Svenke (@anryko)
+requirements: [ botocore, boto3 ]
+options:
+    cluster:
+        description:
+            - The short name or full Amazon Resource Name (ARN) of the cluster that contains the resource to apply attributes.
+        required: true
+    state:
+        description:
+            - The desired state of the attributes.
+        required: false
+        default: present
+        choices: ['present', 'absent']
+    attributes:
+        description:
+            - List of attributes.
+        required: true
+        contains:
+            name:
+                description:
+                    - The name of the attribute. Up to 128 letters (uppercase and lowercase), numbers, hyphens, underscores, and periods are allowed.
+                required: true
+            value:
+                description:
+                    - The value of the attribute. Up to 128 letters (uppercase and lowercase), numbers, hyphens, underscores, periods, at signs (@), forward slashes, colons, and spaces are allowed.
```

Comment:
CI failure due to PEP 8 issue:

```
2017-02-14 22:18:30 ERROR: PEP 8: lib/ansible/modules/cloud/amazon/ecs_attribute.py:52:161: E501 line too long (197 > 160 characters) (current)
```

The PEP 8 tests can be run locally with `make pep8`.